demos/tabs.html

```diff
@@ -0,0 +1,496 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="assets/material-components-web.css.js" charset="utf-8"></script>
+    <style>
+    	.demo-body {
+    	  padding: 24px;
+    	  margin: 0;
+    	  box-sizing: border-box;
+    	}
+
+      .demo-content > button {
+        margin-bottom: 6px;
+      }
+
+      .mdc-theme--dark {
+        background-color: #303030;
+      }
+
+      .mdc-theme--dark h1,
+      .mdc-theme--dark h2,
+      .mdc-theme--dark em,
+      .mdc-theme--dark .panel {
+        color: #a4a4a4;
+      }
+
+      .toggle-rtl {
+        margin-bottom: 32px;
+      }
+
+      #dynamic-demo-toolbar h1 {
+        margin-left: 64px;
+      }
+
+      #dynamic-demo-toolbar .mdc-tabs {
+        position: absolute;
+        bottom: 0;
+        margin-left: 20px;
+      }
+
+      section + section {
+        margin-top: 40px;
+      }
+
+      .panels {
+        padding: 8px;
+        border: 1px solid #ccc;
+        border-radius: 4px;
+        margin-top: 8px;
+      }
+
+      .panel {
+        display: none;
+      }
+
+      .panel.active {
+        display: block;
+      }
+
+      .dots {
+        display: flex;
+        justify-content: flex-start;
+        margin-top: 4px;
+        padding-bottom: 16px;
+      }
+
+      .dot {
+        margin: 0 4px;
+        border-radius: 50%;
+        border: 1px solid #ccc;
+        width: 20px;
+        height: 20px;
+      }
+
+      .dot:last-child {
+        margin-right: 0;
+      }
+
+      .dot.active {
+        background-color: blue;
+        border-color: blue;
+      }
+
+    </style>
+  </head>
+  <body class="mdc-typography">
+    <div class="demo-body">
+
+      <h1>Tabs</h1>
+      <button type="button" class="mdc-button mdc-button--primary" id="toggle-rtl">Toggle RTL</button>
+      <button type="button" class="mdc-button mdc-button--primary" id="toggle-dark-theme">Toggle Dark Theme</button>
+
+    <section>
+      <h2>Basic tabs</h2>
+      <nav id="basic-tabs" class="mdc-tabs">
+        <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+        <a class="mdc-tab" href="#two">Item Two</a>
+        <a class="mdc-tab" href="#three">Three</a>
+        <span class="mdc-tabs__indicator"></span>
+      </nav>
+    </section>
+
+    <section>
+      <h2>CSS-Only Tabs</h2>
+      <nav class="mdc-tabs">
+        <a class="mdc-tab mdc-tab--active" href="#one">Item One</a>
+        <a class="mdc-tab" href="#two">Item Two</a>
+        <a class="mdc-tab" href="#three">Three</a>
+        <span class="mdc-tabs__indicator"></span>
+      </nav>
+    </section>
+
+    <section>
+      <h2>Scrollable tabs</h2>
+
+      <div id="tabs-scroller" class="mdc-tabs-scroller">
+        <div class="mdc-tabs-scroller__indicator mdc-tabs-scroller__indicator--back">
+          <a class="mdc-tabs-scroller__indicator__inner material-icons" href="#" aria-label="Click to scroll back">
```

Comment:
nit: could we change `href="#"` to `href="javascript: void(0)"`? It will help prevent jumping back to the top of the page when I click arrow left and right on tabs